staging_service/autodetect/Mappings.py

```diff
@@ -50,13 +50,27 @@
 attribute_mapping_id = "attribute_mapping"
 escher_map_id = "escher_map"
 
+def _flatten(some_list):
+    # you can do this in a 1 line comprehension but it's unreadable
+    flat_list = []
```

Comment:
You could also do `list(itertools.chain.from_iterable(some_list))`

Reply:
Fixed